docs/HowTo/clusterconnectivity/aiven-kafka-connect-cluster-ssl-protocol.md

```diff
@@ -52,7 +52,7 @@ Kafka Connect service with Klaw using SSL protocol:
    the copy icon that is available on the right-hand side of each
    cluster row.
 
-8. In the `application.properties` file for [cluster-api] (klaw/cluster-api/src/main/resources) module, configure Aiven for Apache Kafka Connect credentials copied from Aiven console:
+8. In the `application.properties` file for [cluster-API] (klaw/cluster-API/src/main/resources) module, configure Aiven for Apache Kafka Connect credentials copied from Aiven console:
```

Comment:
This occurrence should not be capitalised, as it's part of the URL for this link